docker_test.go

```diff
@@ -200,9 +200,11 @@ func TestContainerWithHostNetworkOptions_UseExposePortsFromImageConfigs(t *testi
 		t.Errorf("Expected server endpoint. Got '%v'.", err)
 	}
 
-	_, err = http.Get(endpoint)
+	resp, err := http.Get(endpoint)
 	if err != nil {
 		t.Errorf("Expected OK response. Got '%d'.", err)
+	} else {
+		resp.Body.Close()
 	}
```

Comment:
As in https://github.com/testcontainers/testcontainers-go/pull/1358/files#diff-037b83426e676aa3786c08b334b0db7bb70235792961cbe4ed3340bc0a60a7a8R728
```suggestion
	}
	defer resp.Body.Close()
```

Reply:
I put it into the else branch because we can only call (deferred) resp.body.Close only if err is nil otherwise it would panic. Here error check uses t.Error that does not stop execution of the test unlike t.Fatal in the example you mention.

Reply:
Changed this to defer body close and test response status code like elsewhere.